subprojects/platform-native/src/testFixtures/groovy/org/gradle/nativeplatform/fixtures/AvailableToolChains.java

```diff
@@ -200,7 +200,7 @@ static private ToolChainCandidate findCygwin() {
     static ToolChainCandidate findSwiftc() {
         File compilerExe = new File("/opt/swift/latest/usr/bin/swiftc");
         if (compilerExe.isFile()) {
-            return new InstalledSwiftc("swiftc").inPath(compilerExe.getParentFile());
+            return new InstalledSwiftc("swiftc").inPath(compilerExe.getParentFile(), new File("/usr/bin"));
```

Comment:
Should we also pass the `/usr/bin` path to the other `InstalledSwiftc` below?

Reply:
I'm leaning against it, but I could be convinced.  This usage is with swift in a non-standard location which is a different location than the other tools, so we have to set up the path to find everything.  The second is finding swiftc in the standard system path which should find it in `/usr/bin` (where the other tools are).  I'd hate to change the test fixture such that it could hide a problem in a real-world usage, so I'm inclined to leave it as is and fix it when we actually have a problem with it.